accesscontroltool-package/pom.xml

```diff
@@ -44,6 +69,8 @@
                     <filterSource>src/main/META-INF/vault/filter.xml</filterSource>
                     <properties>
                         <acHandling>merge_preserve</acHandling>
+                        <installPathRegex>/apps/netcentric/actool/.*</installPathRegex>
```

Comment:
Can’t we restrict further? I think only covering the install folder would make sense here

Reply:
The idea was to have everything installed - however you are right, strictly speaking the two bundles should be enough since for the install hook case, the system user config is not needed (runs as same user as the package manager is called with) and hence it should be fine, will test this.

Reply:
As it turns out, `AcHistoryServiceImpl.persistHistory()` is currently not using the session from the package manager (but rather the service session). In a way it is nicer to have the path as it is anway: that way after package is installed in package manager the AC Tool is fully installed with all its bundles and configurations (conceptually better).

Reply:
Made the filter more explicit in ea81c26e8da7369